packages/console/src/components/Table/index.tsx

```diff
@@ -80,21 +93,25 @@ const Table = <
                     </td>
                   </tr>
                 )}
-                {data?.map((row) => (
-                  <tr
-                    key={row[rowIndexKey]}
-                    className={classNames(onClickRow && styles.clickable)}
-                    onClick={() => {
-                      onClickRow?.(row);
-                    }}
-                  >
-                    {columns.map(({ dataIndex, colSpan, className, render }) => (
-                      <td key={dataIndex} colSpan={colSpan} className={className}>
-                        {render(row)}
-                      </td>
-                    ))}
-                  </tr>
-                ))}
+                {data?.map((row) => {
+                  const onClickRow = clickRowHandler?.(row);
+
+                  return (
+                    <tr
+                      key={row[rowIndexKey]}
+                      className={classNames(onClickRow && styles.clickable)}
+                      onClick={() => {
+                        onClickRow?.();
+                      }}
```

Comment:
`onClick = conditional(onClickRow &&  () => clickRowHandler?.(row))`.  If onClickRow is undefined, should not have any onClick event triggered event an empty callback method.

Reply:
Cool, updated.